EventsExpress.Db/Entities/UnitOfMeasuring.cs

```diff
@@ -12,6 +12,8 @@ public class UnitOfMeasuring : BaseEntity
 
         public ICollection<Inventory> Inventories { get; set; }
 
+        public string CategoryOfMeasuring { get; set; }
```

Comment:
String fields in this case are denormalized. Let's create a separate table for categories and link it to entity.